tools/nl/embeddings/build_custom_dc_embeddings.py

```diff
@@ -14,117 +14,140 @@
 """Build embeddings for custom DCs."""
 
 from dataclasses import asdict
+import os
```

Comment:
Is there any unit test to validate this (esp in Keyur's absence I'm not sure of this flow)?

Reply:
I have manually tested this in local and docker env.